### ISSUE TITLE
After new article is published with date in the past, the previous article is hidden from view and publish date is unassigned

### ISSUE THREAD

**Issue by DaDlugosch** [ComfortablyCoding/strapi-plugin-publisher#86]
Hello. I encountered an issue with publishing articles after the update from 1.3.6 to 1.3.7.  When I publish an article with the date in the past, the previous one is hidden from the view (can be shown in the database as a column `is_visible_in_list_view` with value `0`) and the publish date is removed from the database (can be shown as a value `null`). And this is repeated after each publication, only when the article's status is changed from draft to published.

<img width="2046" alt="image" src="https://github.com/ComfortablyCoding/strapi-plugin-publisher/assets/39583615/25299aa8-be66-4869-a392-d387bb17ae66">

**Comment by ComfortablyCoding** [comment]
Did you also update your strapi version as well? 

I would assume it's some regression there first as the only change between v1.3.6 and v1.3.7 was how deps are handled (https://github.com/ComfortablyCoding/strapi-plugin-publisher/compare/1.3.6...1.3.7)

**Comment by DaDlugosch** [comment]
@ComfortablyCoding Yes, I also updated strapi version along with these updates:
<img width="1467" alt="image" src="https://github.com/ComfortablyCoding/strapi-plugin-publisher/assets/39583615/007c465e-0d66-4c15-ab82-0da0d5db875d">

**Comment by DaDlugosch** [comment]
@ComfortablyCoding But you're right, Strapi released a new version and they mentioned some issues with publication (https://github.com/strapi/strapi/releases/tag/v4.11.7).

**Comment by DaDlugosch** [comment]
@ComfortablyCoding Even after I updated the Strapi version, where should be some publication errors fixed, the issue still persists. Also, when I select a date in the publication field, it changes a date to one day before. I attached the video example.

https://github.com/ComfortablyCoding/strapi-plugin-publisher/assets/39583615/108b070f-c6fc-4108-a2aa-5be456d050f0

**Comment by ComfortablyCoding** [comment]
I will check to see if I can reproduce this issue.

If you have a test instance for your setup I would recommend rolling back the changes and then updating each one by one and seeing at what point the error reproduces.

**Comment by DaDlugosch** [comment]
@ComfortablyCoding After further investigation, I found out that the issue was caused by upgrading plugin `@notum-cz/strapi-plugin-content-versioning` from the version `0.4.7` to the version `0.4.9`.
The new issue can be found in the correct repo here: https://github.com/notum-cz/strapi-plugin-content-versioning/issues/104.

Anomalies with date selection still persists, but I think it should be topic for a new issue, not this one.
